core/translate.c

```diff
@@ -1104,7 +1104,14 @@ recreate_app_state_from_ilist(dcontext_t *tdcontext, instrlist_t *ilist, byte *s
                         LOG(THREAD_GET, LOG_INTERP, 2,
                             "recreate_app -- found valid state pc " PFX "\n", answer);
                     } else
-#endif /* X86 */
+#elif defined(AARCH64)
+                    int op = instr_get_opcode(inst);
```

Comment:
This should be up in the cases in `translate_walk_track_post_instr` to avoid `unsupported_mangle` being set in the first place.  Probably the other instrs generated here like xpaci should be in there too so we can confidently generate full state in the middle for asynch translation (synchall for detach, e.g.: don't make it retry b/c it landed on xpaci).

Reply:
Done.